src/text/index.tsx

```diff
@@ -12,6 +12,8 @@ export enum TextDisplayType {
   DISPLAY2 = 'display2',
   SUBHEADER = 'subheader',
   SUBHEADERSTRONG = 'subheaderStrong',
+  TITLE = 'title',
```

Comment:
we removed it from the previous PR. It was supposed to be part of the title. If you need this as a text, maybe this is the title component you are looking for?

Reply:
Actually, I told him to put it back. This `title` display is not used for titles per se but for Item titles. So we need it here

Reply:
Then I maintain what I said from the beginning, we should rename it. If we don't only use it for titles......

Reply:
Well, technically, those are titles, item titles. IMO, the one that needs renaming is `display1`, the one used for the titles/theVoice in Kairos

Reply:
Guys, a code review is not the good place to discuss the naming of the design system. 

My opinion on this is: 
* we should stick to the Pixar naming, 
* Then challenge the naming in Pixar.

But we shouldn't be unsync